hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnection.java

```diff
@@ -262,7 +262,7 @@ private void logClose() {
     }
 
     private Level resolveLogLevelOnClose() {
-        if (!ioService.isActive()) {
+        if (!serverContext.isActive()) {
```

Comment:
doesn't read right, active/inactive context?

Reply:
That can be fixed.

Reply:
```
  @Override
    public boolean isNodeActive() {
        return node.getState() != NodeState.SHUT_DOWN;
    }
```